graphql/models/workspace.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const workspaceSchema = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  color: {
+    type: String
+  },
```

Comment:
```suggestion
  },
  {
    bibliography_style: {
      type: String
    }
  },
```

Ça permettrait de sélectionner au niveau de la revue le style bibliographique, et de le présélectionner en fonction dans la modale Export.

Reply:
Je me demande si c'est pas trop spécifique. Un espace de travail n'est pas forcement lié à une revue et à un style bibliographique en particulier. Cela peut aussi être un espace de travail sur un sujet commun entre des étudiants.

Reply:
Ça dépend du workspace en effet. S'il y a un workspace "Sens Public Montréal", le style d'export par défaut sera toujours celui-là. Et bien entendu l'idée est de laisser le choix dans la modale d'export, mais au point de tout le temps positionner le choix par défaut à celui du workspace.